a10_octavia/common/config_options.py

```diff
@@ -291,11 +291,7 @@
                help=_('Write Memory interval in seconds')),
     cfg.BoolOpt('disable_write_memory',
```

Comment:
I think we should change the option name. Since it not really disable write memory, just disable the periodical write memory mechanism.

Reply:
`use_periodic_write_mem` = 'enable/disable' --- > default (disable) ie controller worker write_memory